feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/MoodHistory.java

```diff
@@ -0,0 +1,35 @@
+package com.example.feelslikemonday.model;
+import java.util.ArrayList;
+import java.util.List;
+
+//The reason why the MoodHistory is an object is so that operations such as sorting and any
+// aggregate operations can be performed here. Should be useful for sorting or filtering.
+//This object composition may be overkill, and might be able extend the ArrayList class.
+
+//Todo: Extend the ArrayList class?
```

Comment:
what would change if it were to extend the ArrayList class?

Reply:
Good point. not a lot. Likely violates liskov.